transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -100,9 +99,7 @@ private B self() {
      * {@link Channel} implementation has no no-args constructor.
      */
     public B channel(Class<? extends C> channelClass) {
-        if (channelClass == null) {
-            throw new NullPointerException("channelClass");
-        }
+        ObjectUtil.checkNotNull(channelClass, "channelClass");
         return channelFactory(new ReflectiveChannelFactory<C>(channelClass));
```

Comment:
@qeesung: nit you can merge the above 2 lines as `checkNotNull` returns `channelClass`.

Reply:
Done